src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -154,6 +159,13 @@ void fillInnerParts() {
             personDetailView = new PersonDetailView(logic.getFilteredPersonList().get(0));
             personDetailViewPlaceholder.getChildren().setAll(personDetailView.getRoot());
         }
+
+        // Displays the first event in the list if exists onto the EventDetailView
+        if (!logic.getFilteredEventList().isEmpty()) {
+            eventDetailView.setEvent(logic.getFilteredEventList().get(0));
+        } else {
+            eventDetailViewPlaceholder.getChildren().clear(); // Display an empty screen if no events
+        }
```

Comment:
This might cause a little problem in the future. Would it better to have this line after the line `eventDetailView = new EventDetailView()`

Reply:
Good point